first pass of adding ADR repo (#1)
mozilla/sre-adrs: decisions/websre/0001-record-architecture-decisions.md

# 1. Record architecture decisions

Date: 2021-04-28
Scope: Web SRE Team

## Status

Proposed

## Context

> We need to record the architectural decisions made on this project.

Reviewer: ```suggestion
We need to record the architectural decisions made for our team.
```
